prog/dftb+/lib_dftbplus/main.F90

```diff
@@ -85,6 +85,12 @@ module dftbp_main
   use dftbp_initprogram, only : TRefExtPot
   use dftbp_qdepextpotproxy, only : TQDepExtPotProxy
   use dftbp_taggedoutput, only : TTaggedWriter
+  use dftbp_rekscommon
+  use dftbp_reksen
+  use dftbp_reksfon
+  use dftbp_reksinterface
+  use dftbp_reksproperty
+  use dftbp_reksvar
```

Comment:
Please create a high level `dftbp_reks` module which exports all the names form those 6 modules. This would clutter the code less and make maintainance easier, if the internal structure of the REKS part changes.

Reply:
@aradi I am trying to create a high level `dftbp_reks` module, but I have no idea for this. I think that the parts in the loop lpSCC_REKS should be moved to the `dftbp_reks` module to rearrange those modules (`dftbp_reksen`, `dftbp_reksfon`, etc). Is there any method to use a high level `dftbp_reks` without moving the SCC iteration loops?

Reply:
For the moment, just create one `dftbp_reks` module (within the `lib_reks`) directory, which just imports those 6 modules and exports the public names:
```
module dftbp_reks
  import dftbp_rekscommon
  [..]
  implicit none
  private

  public :: [Add all public names here]

end module dftbp_reks
```
And then, in `main.F90` you would just only import that single module. (At some point later we may turn it then into a module with submodules).